CRM/Core/Config.php

```diff
@@ -88,13 +88,13 @@ public static function &singleton($loadFromDB = TRUE, $force = FALSE) {
       \Civi\Core\Container::boot($loadFromDB);
 
       if ($loadFromDB && self::$_singleton->dsn) {
-        self::$_singleton->userSystem->postContainerBoot();
```

Comment:
@ufundo is this still needed as part of this PR?

Reply:
Ug no I don't know how that snuck in